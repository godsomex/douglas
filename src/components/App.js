import React from 'react';
import Nav from './Nav';
import Food from './Food';

function App() {
    return (
        <>
            <Nav />
            <Food />
        </>
    );
}

export default App;
