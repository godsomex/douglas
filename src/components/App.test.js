import React from 'react';
import App from './App';
import Nav from './Nav';

import { configure, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';

configure({ adapter: new Adapter() });

describe('My Test Suite', () => {
    it('My Test Case', () => {
        expect(true).toEqual(true);
    });

    it('renders app without crashing', () => {
        const app = mount(<App />);
        expect(app.find(Nav).length).toEqual(1);
    });
});

describe('Nav', () => {
    test('snapshot renders', () => {
        const component = renderer.create(<Nav />);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});

describe('App', () => {
    test('snapshot renders', () => {
        const component = renderer.create(<App />);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
