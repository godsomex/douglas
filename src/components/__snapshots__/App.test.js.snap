// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot renders 1`] = `
Array [
  <div
    className="css-1ce784x"
  >
    <nav>
      <ul
        className="drop"
      >
        <input
          className="menu-btn"
          id="menu-btn"
          type="checkbox"
        />
        <label
          className="menu-icon"
          htmlFor="menu-btn"
        >
          <span
            className="nav-icon"
          />
        </label>
        <li
          className="logo"
        >
          <a
            href="#"
          >
            TechFun
          </a>
        </li>
        <li
          className="item"
        >
          <a
            href="#"
          >
            Home
          </a>
        </li>
        <li
          className="item"
        >
          <a
            href="#"
          >
            About
          </a>
        </li>
        <li
          className="item"
        >
          <a
            href="#"
          >
            Services
          </a>
        </li>
        <li
          className="item button"
        >
          <a
            href="#"
          >
            Log In
          </a>
        </li>
        <li
          className="item button secondary"
        >
          <a
            href="#"
          >
            Sign Up
          </a>
        </li>
      </ul>
    </nav>
  </div>,
  <div
    className="css-o6mf5o"
  >
    <div
      className="css-qyry97"
    >
      <h5
        className="css-1cctmdv"
      >
        <input
          name="vegetable"
          onChange={[Function]}
          type="checkbox"
        />
        <label
          htmlFor="veg"
        >
          Vegetable
        </label>
      </h5>
      <h5
        className="css-xivvka"
      >
        <input
          name="meat"
          onChange={[Function]}
          type="checkbox"
        />
        <label
          htmlFor="meat"
        >
          Meat
        </label>
      </h5>
      <h5
        className="css-1750ecm"
      >
        <input
          name="fruit"
          onChange={[Function]}
          type="checkbox"
        />
        <label
          htmlFor="fruit"
        >
          Fruit
        </label>
      </h5>
    </div>
    <div
      className="css-1nakvlf"
    >
      <div
        className="contain"
      >
        <ul
          className="responsive-table"
        >
          <li
            className="table-row"
          >
            <div
              className="cols cols-2"
            >
              Filter
            </div>
            <div
              className="cols cols-2"
              data-label="Vegetable"
            >
              <input
                name="vegetable"
                onChange={[Function]}
                type="checkbox"
              />
            </div>
            <div
              className="cols cols-2"
              data-label="Meat"
            >
              <input
                name="meat"
                onChange={[Function]}
                type="checkbox"
              />
            </div>
            <div
              className="cols cols-2"
              data-label="Fruit"
            >
              <input
                name="fruit"
                onChange={[Function]}
                type="checkbox"
              />
            </div>
          </li>
        </ul>
      </div>
    </div>
    <main
      className="main-area"
    >
      <div
        className="centered"
      >
        <section
          className="cards"
        />
      </div>
    </main>
  </div>,
]
`;

exports[`Nav snapshot renders 1`] = `
<div
  className="css-1ce784x"
>
  <nav>
    <ul
      className="drop"
    >
      <input
        className="menu-btn"
        id="menu-btn"
        type="checkbox"
      />
      <label
        className="menu-icon"
        htmlFor="menu-btn"
      >
        <span
          className="nav-icon"
        />
      </label>
      <li
        className="logo"
      >
        <a
          href="#"
        >
          TechFun
        </a>
      </li>
      <li
        className="item"
      >
        <a
          href="#"
        >
          Home
        </a>
      </li>
      <li
        className="item"
      >
        <a
          href="#"
        >
          About
        </a>
      </li>
      <li
        className="item"
      >
        <a
          href="#"
        >
          Services
        </a>
      </li>
      <li
        className="item button"
      >
        <a
          href="#"
        >
          Log In
        </a>
      </li>
      <li
        className="item button secondary"
      >
        <a
          href="#"
        >
          Sign Up
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
